[2025-01-17T18:17:15.515+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T18:17:15.532+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:17:13.263649+00:00 [queued]>
[2025-01-17T18:17:15.540+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:17:13.263649+00:00 [queued]>
[2025-01-17T18:17:15.540+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T18:17:15.555+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_api_python> on 2025-01-17 18:17:13.263649+00:00
[2025-01-17T18:17:15.563+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3262) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T18:17:15.564+0000] {standard_task_runner.py:72} INFO - Started process 3266 to run task
[2025-01-17T18:17:15.565+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_ELT_PostgreSQL', 'get_api_python', 'manual__2025-01-17T18:17:13.263649+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/ETL_PostgreSQL.py', '--cfg-path', '/tmp/tmpt6iyhgo1']
[2025-01-17T18:17:15.566+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask get_api_python
[2025-01-17T18:17:15.609+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_ELT_PostgreSQL.get_api_python manual__2025-01-17T18:17:13.263649+00:00 [running]> on host 59ff16f1d4fb
[2025-01-17T18:17:15.989+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_ELT_PostgreSQL' AIRFLOW_CTX_TASK_ID='get_api_python' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T18:17:13.263649+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T18:17:13.263649+00:00'
[2025-01-17T18:17:15.990+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-17T18:17:15.990+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-17T18:17:15.990+0000] {logging_mixin.py:190} INFO - Current task name:get_api_python state:running start_date:2025-01-17 18:17:15.532811+00:00
[2025-01-17T18:17:15.990+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL and current dag run status:running
[2025-01-17T18:17:15.991+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T18:17:16.734+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T18:17:16.746+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T18:17:16.747+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_ELT_PostgreSQL, task_id=get_api_python, run_id=manual__2025-01-17T18:17:13.263649+00:00, execution_date=20250117T181713, start_date=20250117T181715, end_date=20250117T181716
[2025-01-17T18:17:16.762+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-17T18:17:16.763+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-17T18:17:16.763+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL queued_at:2025-01-17 18:17:13.274876+00:00
[2025-01-17T18:17:16.764+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-17T18:17:16.783+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T18:17:16.813+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-17T18:17:16.816+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
